draft-irtf-cfrg-hpke.md

```diff
@@ -489,10 +504,12 @@ context. The key schedule inputs are as follows:
 * `mode` - A one-byte value indicating the HPKE mode, defined in {{hpke}}.
 * `zz` - A KEM shared secret generated for this transaction
 * `info` - Application-supplied information (optional; default value
-  "")
+  "") of maximum length 65535 bytes.
```

Comment:
I actually don't need a length limit on HKDF `info` inputs anymore in order to avoid allocation. Your call if you want to keep it or not.

Reply:
I inherited this from length restrictions on the TLS Exporter API.